python/ray/ml/result.py

```diff
@@ -9,6 +9,10 @@ class Result:
     checkpoint: Optional[Checkpoint]
     metrics: Any
 
+    def __init__(self, metrics: Any, checkpoint: Optional[Checkpoint]):
+        self.metrics = metrics
+        self.checkpoint = checkpoint
+
```

Comment:
Let's remove the class-level checkpoint and metrics variable then

Reply:
Removed!